Mathlib/Combinatorics/SimpleGraph/Walk.lean

```diff
@@ -809,6 +809,23 @@ theorem nodup_tail_support_reverse {u : V} {p : G.Walk u u} :
     ← getVert_eq_support_getElem? _ (by rw [Walk.length_support]; cutsat)]
   aesop
 
+theorem edges_eq_support {u v : V} {p : G.Walk u v} :
+    p.edges = (p.support.zip p.support.tail).map (fun x ↦ s(x.1, x.2)) := by
```

Comment:
It looks like `List.zipWith` would make more sense to use here?

Reply:
I agree, done